### ISSUE TITLE
All properties are optional

### ISSUE THREAD

**Issue by danmarshall** [mstssk/sw2dts#17]
Hello, thanks for the cool project. In my scenario I don't need the web client code, so sw2dts provided the cleanest output.

I noticed that all properties are optional - with the question mark. Is there any way to disable this?

**Comment by bricka** [comment]
I thought this was happening too, but actually, properties are optional by default in Swagger.  So you need to mark them as required.

For example:

```yaml
Point:
  type: object
  properties:
    x:
      type: number
    y:
      type: number
```

produces:

```typescript
export interface Point {
    x?: number;
    y?: number;
}
```

But if you mark them as required:

```yaml
Point:
  type: object
  properties:
    x:
      type: number
    y:
      type: number
  required:
    - x
    - y

```

produces:

```typescript
export interface Point {
    x: number;
    y: number;
}
```

**Comment by mstssk** [comment]
@danmarshall @bricka Thank you for your notice and a advice!
Properties need `required` flag to be no-optional properties as @bricka says.

However, I found a bug that is 'required' flag in query parameters was missed with '--with-query' option.
So I fix it (#18) and released v2.4.0.
https://github.com/mstssk/sw2dts/blob/master/CHANGELOG.md#240-2017-06-08

Thanks!
